### ISSUE TITLE
'make deb' fails when git is not installed

### ISSUE THREAD

**Issue by agaffney** [ansible/ansible#13329]
I tried building a package for ansible 2.0.0-rc1 inside a Docker container, but it failed with an odd error.

```
# make DEB_DIST=trusty deb
...
parsechangelog/debian: warning:     debian/changelog(l1): version '2.0.0-0.git201511270000''~trusty' is invalid: version number contains illegal character `''
LINE: ansible (2.0.0-0.git201511270000''~trusty) trusty; urgency=low
debuild: unknown dpkg-buildpackage/debuild option: --source-option=-I
 dpkg-buildpackage -rfakeroot -D -us -uc --source-option=-I -b
dpkg-buildpackage: warning: using a gain-root-command while being root
parsechangelog/debian: warning:     debian/changelog(l1): version '2.0.0-0.git201511270000''~trusty' is invalid: version number contains illegal character `''
LINE: ansible (2.0.0-0.git201511270000''~trusty) trusty; urgency=low
dpkg-buildpackage: error: version number contains illegal character `''
dpkg-buildpackage: source package ansible
dpkg-buildpackage: source version 2.0.0-0.git201511270000''~trusty
debuild: fatal error at line 1364:
dpkg-buildpackage -rfakeroot -D -us -uc --source-option=-I -b failed
make: *** [deb] Error 29
```

After digging a bit, I found the following line in the Makefile:

```
47:GITINFO = ''
```

It appears that `make` requires double quotes instead of single quotes. The above line gets interpreted as assigning a string with two literal single quotes rather than an empty string.

I wondered why nobody had ever hit this bug. It appears that this can only happen when you build without `git` installed. I was building inside a Docker container from a github release tarball, so there was no need to install `git`.

**Comment by agaffney** [comment]
It should be noted that changing that Makefile line above to the following fixes the issue.

```
GITINFO = ""
```

**Comment by bcoca** [comment]
fixed via b0e5556d9aa819a9bbcabfdbe7dca62c23cc6828
